web/docs/tql2/operators/chart_area.md

```diff
@@ -0,0 +1,56 @@
+# chart_area
+
+Chart values on a X-Y area chart.
+
+```tql
+chart_area x=any, y=any, [x_min=any, x_max=any, resolution=duration, x_log=bool, y_log=bool, group=any, position=string]
+```
+
+## Description
+
+Chart values on X-Y area chart.
+
+### `x = any`
+
+The values to plot on the X-axis.
+
+### `y = any`
+
+The values to plot on the Y-axes. The values are grouped for each unique `x`
+with the given aggregation if specified.
+
+If specified as a record, the field names are used to label the Y-axes.
+
+### `x_min = any (optional)`
+
+If specified, only charts events where `x > from`.
+
+### `x_max = any (optional)`
+
+If specified, only charts events where `x < to`.
+
+### `x_log = bool (optional)`
+
+If `true`, use a logarithmic scale for the X-axis.
+
+Defaults to `false`.
+
+### `y_log = bool (optional)`
+
+If `true`, use a logarithmic scale for the Y-axis.
+
+Defaults to `false`.
+
+### `group = any (optional)`
+
+Optional field to group the aggregations with.
```

Comment:
This is hard to understand, in particular when there are no "aggregations" from the POV of the user (even if there technically always are aggregations).

Reply:
@tobim suggested naming the option `color` which I think is also used by other charting libraries?

Reply:
Sticking with `group` now, and we'll address the docs in a follow-up.